parsec/event_bus.py

```diff
@@ -15,40 +33,54 @@ class MetaEvent(Enum):
     EVENT_DISCONNECTED = "event.disconnected"
 
 
+CustomEvent = TypeVar("CustomEvent", bound=Enum)
+AllEvent = Union[MetaEvent, CustomEvent]
+
+
+class EventCallback(Protocol):
+    def __call__(self, event: Enum, **kwargs: Any) -> None:
```

Comment:
One alternative to `Any` is to use `object`. It guarantees that those objects are not used for anything, and simply passed around. This way, it's up to the event handlers to cast the values they get from event bus according to a given "protocol" (e.g. keyword argument "x" of event "y" is supposed to be of type "z"). 

This would allow mypy to catch more errors since casting has to be added explicitly by the devs, instead of the implicitness of `Any`.

Reply:
I didn't know that, seems pretty good to me ;-(

Reply:
cf7d9408ba8492afd17f56a7de61931e5780c0c1